### PR TITLE
improve: test cases

### PR DESCRIPTION
**Pull request by theBeardA**

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
           - name: check-fmt-clippy
             task: "ci"
           - name: test
-            task: "test-network"
+            task: "test"
```
Comment (ozwaldorf) [inline .github/workflows/rust.yml:22]: can remove the make entry `test-network` since we're not using it anymore

**Comment by ozwaldorf** [comment]
service test put and get is failing:
image

File: Cargo.toml [hunk 2]
```diff
@@ -26,7 +26,7 @@ async-fs = "1.6.0"
 async-trait = "0.1.60"
 axum = { version = "0.6.1", features = ["multipart", "headers"] }
 axum-server = { version = "0.4.4", features = ["tls-rustls"] }
-base64 = "0.20.0"
```
Comment (b0xtch) [inline Cargo.toml:29]: We going down down down?

Comment (theBeardA) [reply Cargo.toml:29]: yes, I upgraded that in workspace dependency branch and it broke few things. I will open another issue to upgrade this.

**Comment by b0xtch** [comment]
https://github.com/fleek-network/ursa/blob/b92fc66eddf292adb874b39c3b546c38e3dd5637/.github/workflows/rust.yml#L22

Could we also update the test here to all now 🙏

**Review by b0xtch — APPROVED** [review]

**Pull request merged**